app/controllers/container_controller.rb

```diff
@@ -48,6 +48,9 @@ def x_button
   # Container show selected, redirect to proper controller
   def show
     record = Container.find_by_id(from_cid(params[:id]))
+    if @display == "timeline"
```

Comment:
> @dkorn i think you can add a spec test for show method to make sure it renders a correct view and doesn't blow up, look at example here: https://github.com/ManageIQ/manageiq/blob/master/spec/controllers/container_group_controller_spec.rb#L15

After discussing this with @simon3z we've agreed to exclude the UI part for container events and timeline from this PR, therefore I don't think that adding a spec test is needed here. @h-kataria please correct me if I'm wrong.